arroyo-sql/src/expressions.rs

```diff
@@ -1330,7 +1340,16 @@ impl SortExpression {
     }
 
     pub fn to_syn_expr(&self) -> syn::Expr {
-        let value_expr = self.value.to_syn_expression();
+        let value = if self.value.return_type().is_float() {
+            Expression::WrapType(WrapTypeExpression::new(
```

Comment:
We don't need to dip into Expression, could just inline the syn stuff here, but not a big deal.

Reply:
Yeah, I'm imagining we'll have other cases like this and it's worth having the slightly more general solution